packages/opentelemetry-core/src/trace/globaltracer-utils.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as types from '@opentelemetry/types';
+import { GlobalTracerDelegate } from './GlobalTracerDelegate';
+
+const globalTracerDelegate = new GlobalTracerDelegate();
+
+/**
+ * Set the current global tracer. Returns the initialized global tracer
+ */
+export function initGlobalTracer(tracer: types.Tracer): types.Tracer {
```

Comment:
If we have the different tracer packages export a singleton tracer instance that the user can import, and expose configuration options via a `start` method, then can we remove `initGlobalTracer` and `getTracer`?

Reply:
I think it should stay for consistency since the spec is calling for it and other languages will also implement `getTracer`.

Reply:
OK, SGTM.